k9mail/src/main/java/com/fsck/k9/helper/K9AlarmManager.java

```diff
@@ -0,0 +1,66 @@
+package com.fsck.k9.helper;
+
+import android.annotation.TargetApi;
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.os.Build;
+import android.os.PowerManager;
+
+public class K9AlarmManager {
+    private AlarmManager mAlarmManager;
```

Comment:
Please check the [code style](https://github.com/k9mail/k-9/wiki/CodeStyle) document. New code shouldn't use the "m"  prefix.